src/dmd/backend/dwarfeh.d

```diff
@@ -360,8 +360,8 @@ else
         et.writeByte(0);
 
     /* Write out Types Table in reverse */
-    Symbol **typesTable = sfunc.Sfunc.typesTable;
-    for (int i = cast(int)sfunc.Sfunc.typesTableDim; i--; )
+    auto typesTable = sfunc.Sfunc.typesTable[];
+    for (int i = cast(int)typesTable.length; i--; )
```

Comment:
`foreach_reverse`?

Reply:
I'm not a big fan of foreach_reverse, even though it was my idea.

Reply:
Lol